src/index.ts

```diff
@@ -185,10 +212,6 @@ const init = async (mod: Spotfire.Mod) => {
 
         lastDataview = dataView;
 
-        const wordsAxisMeta = await mod.visualization.axis("Words");
-        const fontSizeAxisMeta = await mod.visualization.axis("Font size");
-        const colorAxisMeta = await mod.visualization.axis("Color");
```

Comment:
These were moved to the main reader because they would trigger three serial network requests each time any other readable was updated. Directly reading a readable like this bypasses the subscription mechanism of a reader.